docusaurus/docs-internals/editor-controls/data-controls/gallery.md

```diff
@@ -20,26 +20,69 @@ Example of `gallery` with uploaded images:
 | `helper?.content`       | `string`                                                                                                                                                                      |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                 |
 | `helper?.position`      | `"top-start" \| "top" \| "top-end" \| "right-start" \| "right" \| "right-end" \| "bottom-end" \| "bottom" \| "bottom-start" \| "left-end" \| "left" \| "left-start" \| "top"` | `"top"` | Specifies the position of the tooltip relative to the helper icon                                                                                                                                                                 |
+| `states?`          | `Array<State>`                                                                                                                                                                             | [`"normal"`] | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination)                                                                                         |
 | `config?.canDeleteLast` | `boolean`                                                                                                                                                                     |  `true`  | If provided and set to `false`, the user cannot delete the last item in the gallery.                                                                                                                                              |
+| `default?`         | `Default`                                                                                                                                                                                  |      -       |<b>`Default: { value: string; }`</b><br/><br/>The default control value should be a `JSON.stringify` that represents an array of objects, each object describing an image. <br/> <br/> <b>`Image: { id: number; uid: string; fileName: string; width: number; height: number; }`</b> <br/> <br/> `id` - The index of the image in the gallery <br/> `uid` - The unique identifier of the image<br/>`fileName` - The unique identifier of the image<br/>`width` - The width of the image<br/>`height` - The height of the image<br/>|
```

Comment:
change "The default control value should be a JSON.stringify that represents an array of objects, each object describing an image." to "The default control value should be a `JSON.stringify` of `Image[]`"